packages/frontend/src/components/NavBar/MenuItems.tsx

```diff
@@ -20,6 +20,14 @@ export interface IMenuItem {
   children?: any;
 }
 
+const hasWhitespace = (name: string) => {
+  return /\s/.test(name);
+};
+
+export const getRouterName = (name: string) => {
+  return hasWhitespace(name) ? name.split(' ').join('-').toLocaleLowerCase() : name.toLocaleLowerCase();
```

Comment:
Could this be simplified to using `.replace`? Could also skip the ternary and just do that, because you would/could be using the same regex there. And to top this off, as this is a single line of code, this could skip the {}.